cargo-quickinstall/src/main.rs

```diff
@@ -285,14 +286,21 @@ fn install_crate(crate_name: &str, version: &str, target: &str) -> Result<(), In
 }
 
 fn main() -> Result<(), Box<dyn std::error::Error + Send + Sync + 'static>> {
-    let args = std::env::args().collect::<Vec<_>>();
-    let crate_name = if let Some(true) = args.get(1).map(|a| a == "quickinstall") {
-        args.get(2)
```

Comment:
I should have put a comment here. This hack is to handle how cargo calls out to subcommands. I did a vaguely equivalent thing in my attempt at this, over at:

https://github.com/alsuren/cargo-quickinstall/pull/61/files#diff-30cdea9081430e1fcfa8eea480f74e13b0cb145aaea84f5809a778db1c5cc08cR29

(I'm not sure what the best way to do this is. I might have to ask the maintainer of pico-args about it)